src/index.html

```diff
@@ -20,8 +20,504 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__link"
+      >
+        <img
+          class="logo"
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="nav header__nav">
```

Comment:
This CSS is not following BEM best practices (which sometimes are counterintuitive, but they do pay off! remember that!)
![image](https://github.com/mate-academy/layout_catalog/assets/67425140/255ef822-fd64-4488-ad9b-0fe27c928937)
Try reading this section of the BEM document: https://en.bem.info/methodology/quick-start/#nesting-1